src/components/SearchResults.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Masonry from "react-masonry-css";
+import "../styles/searchResults.css";
+
+const SearchResults = ({ results }) => {
+  if (!results.length) {
+    return <p>No results</p>;
+  }
+
+  const breakpointColumnsObj = {
+    default: 4,
+    1100: 3,
+    700: 2,
+    500: 1,
+  };
+
+  return (
+    <Masonry
+      breakpointCols={breakpointColumnsObj}
+      className="masonry-grid"
+      columnClassName="masonry-grid_column"
+    >
+      {results.map((imageUrl, index) => (
+        <img key={index} src={imageUrl} alt="NASA" className="image" />
```

Comment:
This alt tag isnt all that helpful. Their purpose is to describe what an image is for people using a screen reader, in this case all the images will have this generic label that doesnt help describe anything really. A good next step for future development would be to see if you can get the alt tag for each image from the API and attach it to its image